### PR TITLE
Initial ember setup for the opensentinel ember app

### PR DESCRIPTION
**Pull request by marvinpinto**
It doesn't do very much at the moment, and that's okay :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+* `ember server`
+* Visit your app at [http://localhost:4200](http://localhost:4200).
+
+### Code Generators
+
+Make use of the many generators for code, try `ember help generate` for more details
+
+### Running Tests
+
+* `ember test`
```
Comment (des4maisons) [inline README.md]: `make test`?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "osl-ui",
+  "version": "0.0.0",
+  "description": "Small description for osl-ui goes here",
```
Comment (des4maisons) [inline package.json]: fill in? :)

**Comment by des4maisons** [comment]
awesome :ramen:

**Comment by marvinpinto** [comment]
@des4maisons I totally agree with you :+1: 

I was thinking we could flesh out the contents of that readme as we make progress on the interface. What do you think?

(what you see there is essentially the output of doing an `ember init` -- 90dbab70e9b4773b8681d9359fdc0480f2147c5b)

**Comment by margueritepd** [comment]
Sounds good to me :partly_sunny:

**Pull request merged**